File: schedule/src/main/java/net/yan100/compose/schedule/properties/XxlJobAutoConfigurationProperties.java
package net.yan100.compose.schedule.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * xxl-job 配置项
 *
 * @author t_teng
 * @since 2023-03-25
 */
@Deprecated
@Data
@ConfigurationProperties(prefix = "compose.schedule.xxl-job")
public class XxlJobAutoConfigurationProperties {
  String adminAddress = "http://localhost/xxl-job-admin";
  String accessToken = "default_token";
}

File: depend/xxl-job/src/main/kotlin/net/yan100/compose/schedule/properties/XxlJobAutoConfigurationProperties.kt
package net.yan100.compose.schedule.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.NestedConfigurationProperty

private const val PREFIX = "compose.schedule"

/**
 * xxl-job 配置项
 *
 * @author t_teng
 * @since 2023-03-25
 */
@Deprecated("")
@ConfigurationProperties(prefix = "$PREFIX")
data class XxlJobAutoConfigurationProperties(
  var adminAddress: String = "http://localhost/xxl-job-admin",
  var accessToken: String = "default_token",
  @NestedConfigurationProperty
  var executor: XxlJobExecutorAutoConfigurationProperties =
    XxlJobExecutorAutoConfigurationProperties(),
)
